pkg/skuba/actions/auth/login.go

```diff
@@ -76,30 +85,24 @@ func Login(cfg LoginConfig) (*api.Config, error) {
 		return nil, errors.Wrapf(err, "auth failed")
 	}
 
-	url, err := url.Parse(cfg.DexServer)
-	if err != nil {
-		return nil, errors.Wrapf(err, "parse url")
-	}
-	apiserverURL = fmt.Sprintf("https://%s:6443", url.Hostname()) // Guess kube-apiserver on port 6443
-
 	// fill out clusters
 	kubeConfig := clientcmdapi.NewConfig()
-	kubeConfig.Clusters[cfg.ClusterName] = &api.Cluster{
-		Server:                   apiserverURL,
+	kubeConfig.Clusters[cfg.ClusterName] = &clientcmdapi.Cluster{
+		Server:                   fmt.Sprintf("%s://%s:%s", defaultScheme, url.Hostname(), defaultAPIServerPort), // Guess kube-apiserver on port 6443
```

Comment:
I assume there will be another PR for new option to specify the endpoint of API server in the future.

Reply:
Let's merge this after RC1.

Reply:
Correct!